en/docs/install-and-setup/setup/api-controller/managing-microgateways/managing-microgateways-with-ctl.md

```diff
@@ -0,0 +1,293 @@
+
+    !!! info
+        **Flags:**
+        
+        -    Mandatory :     
+            `--host` or `-c` : The microgateway adapter host url with port  
+
+    !!! example
+        ```bash
+        apictl mg add env prod --host  https://localhost:9443 
+        ```
+
+## Log in to a Microgateway Environment
+
+Log in will get an access token from the microgateway adapter and store it for further commands.
+
+-   **Command**
+    ```go
+    apictl mg login <environment-name> -k
+    ```
+    ```go
+    apictl mg login <environment-name> -u <username> -k
+    ```
+    ```go
+    apictl mg login <environment-name> -u <username> -p <password> -k
+    ```
+    ```go
+    cat <file-with-the-password> | apictl mg login <environment-name> -u <username> --password-stdin -k
+    ``` 
+
+    !!! tip
+        If you run `apictl mg login <environment-name>` you are prompted to provide both the username and the password.
+        If you run `apictl mg login <environment-name> --username <username>`, you are prompted to provide the password.
+        If you include the flag `--password-stdin`, you can provide the password in a file as shown above.
+
+    !!! info
+        **Flags:**
+        
+        - Optional :     
+            `--username` or `-u` : Username for login  
+            `--password` or `-p` : Password for login     
+            `--password-stdin` : Get password from stdin  
+
+    !!! example
+        ```bash
+        apictl mg login dev -k
+        ```
+        ```bash
+        apictl mg login dev -u admin -k
+        ```
+        ```bash
+        apictl mg login dev -u admin -p admin -k
+        ```
+        ```bash
+        cat ~/.mypassword | apictl mg login dev -u admin --password-stdin
+        ```
+-   **Response**
+
+        ``` bash tab="Response Format"
+        Successfully logged into Microgateway Adapter in environment: <environment-name>
+        ```
+
+        ```bash tab="Example Response"
+        Successfully logged into Microgateway Adapter in environment: dev
+        ```
+    
+    !!! warning
+        Using `--password` in CTL is not secure. Instead, you can use `--password-stdin` to include password file or provide the password when prompted. 
+
+## Deploy an API
+
+In order to deploy, let's first create an API project. Follow the steps in [Initialize an API Project]({{base_path}}/install-and-setup/setup/api-controller/managing-apis-api-products/importing-apis-via-dev-first-approach/#initialize-an-api-project) to create an API project using **apictl**.
+
+An API project can be deployed to a microgateway using the following commands.
+
+-   **Command**
+    ```go
+    apictl mg deploy api -e <environment-name> -f <api-project-file-path> -k
+    ```
+    ```go
+    apictl mg deploy api ---environment <environment-name> --file <api-project-file-path> -k
+    ```
+
+    !!! tip
+        In an API project the API name and the API version given in the api.yaml will be taken as the name and version of the API once deployed.
```

Comment:
```suggestion
        In an API project the API name and the API version given in the `api.yaml` will be taken as the name and version of the API once deployed.
```
It is a best practice if the file names such as api.yaml can be mentioned as above within text.

Reply:
Thanks, updating this. Will remember for the others.